libraries/chain/steem_evaluator.cpp

```diff
@@ -196,13 +196,12 @@ namespace golos { namespace chain {
         }
 
         void account_create_with_delegation_evaluator::do_apply(const account_create_with_delegation_operation& o) {
-            ASSERT_REQ_HF(STEEMIT_HARDFORK_0_18__535, "Account creation with delegation");
+            ASSERT_NEED_HF(STEEMIT_HARDFORK_0_18__535, "Account creation with delegation");
```

Comment:
What is wrong with ASSERT_REQ_HF? That was at least a correct word usage.

Reply:
`REQ` can be ambiguous (is it "required" or "request"?).
But if it fails to add clarity, I'll revert.